packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/ola-step-checker/ola-step-checker.controller.js

```diff
@@ -49,4 +31,11 @@ export default class {
       maximumSignificantDigits: 1,
     }).format(price);
   }
+
+  hasBandwidthExtension() {
+    const specs = filter(this.ola.availableModes, {
+      interfaces: [{ count: 4, aggregation: true }],
+    });
+    return specs.length > 0;
```

Comment:
You can directly return `this.ola.availableModes.some(() => // Conditions here)`. (Or with `_some` if you use property shorthand)